src/angularCore.ts

```diff
@@ -0,0 +1,74 @@
+import type {
+  Address,
+  Fetcher,
+  FetcherDeclaration,
+  FetcherDeclarationEthers,
+  FetcherDeclarationEthereum,
+  FetcherProp,
+  NftMetadata,
+} from "./types"
+import type { EthersFetcherConfig } from "./fetchers/ethers/types"
+import type { EthereumFetcherConfig } from "./fetchers/ethereum/types"
+
+import ethersFetcher from "./fetchers/ethers"
+import ethereumFetcher from "./fetchers/ethereum"
+
+const NFT_METADATA_DEFAULT = {
+  name: "",
+  description: "",
+  image: "",
+} as NftMetadata
+
+export class NftProviderAngular {
```

Comment:
I’m not sure how we could name this class if we remove the `Angular`. [`NftProvider`](https://github.com/spectrexyz/use-nft/blob/19af6ac3555fd40870eedfe4b2dfb92cdbf36c87/src/core.tsx#L72) is already exported, and the term “provider” is something specific in React so I think it could be a good idea to avoid it here. What about something like `FetchWrapper`?